modules/sdk/project-templates/project-templates-war-cdi-bean-portlet/src/main/resources/archetype-resources/src/main/java/portlet/__className__Portlet.java

```diff
@@ -0,0 +1,39 @@
+package ${package}.portlet;
+
+import ${package}.constants.${className}PortletKeys;
+
+import com.liferay.bean.portlet.LiferayPortletConfiguration;
+
+import javax.inject.Inject;
+
+import javax.portlet.PortletConfig;
+import javax.portlet.annotations.LocaleString;
+import javax.portlet.annotations.PortletConfiguration;
+import javax.portlet.annotations.RenderMethod;
+
+/**
+ * @author ${author}
+ */
+@PortletConfiguration(
+	portletName = ${className}PortletKeys.${className},
+	title = @LocaleString(value = ${className}PortletKeys.${className}))
```

Comment:
Add a new line before the ending right parenthesis to match the existing pattern in portal.